subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolutionstrategy/DefaultResolutionStrategy.java

```diff
@@ -71,6 +71,7 @@
     private final VcsResolver vcsResolver;
     private final ComponentSelectorConverter componentSelectorConverter;
     private final DependencyLockingProvider dependencyLockingProvider;
+    private final DependencyLockingProvider noLockingDependencyLockingProvider;
```

Comment:
Type and variable name don't align. Should they?

Reply:
This is gone, back to having a no-op singleton